js/ui/scheduler/ui.scheduler.appointment_tooltip.js

```diff
@@ -48,7 +51,8 @@ var appointmentTooltip = {
                 at: "top",
                 of: $appointment,
                 boundary: isAllDay ? instance.$element() : instance.getWorkSpaceScrollableContainer(),
-                collision: "fit flipfit"
+                collision: "fit flipfit",
+                offset: { x: 0, y: this.instance.option("_appointmentTooltipVerticalOffset") }
```

Comment:
I think better to use  _appointmentTooltipOffset and set it to { x: 0, y: 11 } for material

Reply:
fixed